ci/scripts/check-module-ids.sh

```diff
@@ -23,6 +23,7 @@ query_elfs='
         )
     )'
 target_file=$(mktemp)
+trap 'rm -f "$target_file"' EXIT
```

Comment:
Oh wow! I thought that the single quotes would break expansion here, but I just tried it locally and it works. Magic! :-D

Reply:
I admit I was also dubious so I run a test on my machine and I was also surpised that it works:
```sh
touch hello
touch world
touch "hello world"
file_name="hello world"
trap 'rm "$file_name"' EXIT
```
I don't think I really understand bash...

Reply:
Ohh! I've just understood! The point is that the single quotes stop the expansion of the variable while we're installing the trap, so the trap gets installed as "rm $file_name" (with the dollar). But then the expansion of "$file_name" happens when we actually run the handler.

So this works as it should:
```
touch hello
x=world
trap 'rm $x' EXIT
x=hello
exit
```

Reply:
Yes exactly

Reply:
⬛🪄